packages/server/postgres/queries/getIntegrationProvidersByIds.ts

```diff
@@ -14,6 +14,13 @@ interface IntegrationProviderWebhook extends IntegrationProviderActive {
   webhookUrl: string
 }
 
+interface IntegrationProviderOAuth1 extends Omit<IntegrationProviderActive, 'scope'> {
+  authStrategy: 'oauth1'
+  scope: Omit<IntegrationProviderScopeEnum, 'global'>
```

Comment:
+1 i suppose there could be an integration that uses oauth1 that is global? i think confluence cloud might, but we can cross that bridge when it comes. safe for now!